web/index.html

```diff
@@ -9,6 +9,8 @@
     <meta name="robots" content="index, follow">
     <meta name="viewport" content="minimal-ui, width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
     <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
```

Comment:
Will this always be good for the game? If not, add a TODO item to remove in prod plese.

Reply:
@utgarda , 
yes. removing  completely caching. 
`iframe` where we insert client game's frontend should never contain caches from client (always serve a non-cached version).
at the `platform-frontend` same -> added a timestamp to url to not allow to cache previous versions (added a timestamp arg in the URL).
only fresh non-cached pages